.gitignore

```diff
@@ -18,5 +15,20 @@ install_manifest.txt
 .project
```

Comment:
maybe we could create a repository for Cadence's code and use it as an submodule. i think it would be cleaner.